### PR TITLE
Parent attribute

### PR DESCRIPTION
**Pull request by codinglawyer**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -86,6 +86,7 @@ const Popover = createClass({
     enterExitTransitionDurationMs: T.number,
     className: T.string,
     style: T.object,
+    parent: T.object,
```
Comment (iakovmarkov) [inline src/index.js]: `T.object`? I'd rather use `T.node`, it's a DOM node after all, not just some Object.

File: src/index.js [hunk 2]
```diff
@@ -375,11 +385,13 @@ const Popover = createClass({
   untrackPopover () {
     clearInterval(this.checkLayoutInterval)
     this.frameEl.removeEventListener(`scroll`, this.onFrameScroll)
+    this.props.parent && (document.querySelector('.AutocompleteField-body').style.display = 'none')
```
Comment (iakovmarkov) [inline src/index.js]: Nope. Please, no hardcoded stuff like this. Find a better way to hide the element on untracking

File: src/index.js [hunk 3]
```diff
@@ -375,11 +385,13 @@ const Popover = createClass({
...
     resizeEvent.off(this.frameEl, this.onFrameResize)
     resizeEvent.off(this.containerEl, this.onPopoverResize)
     resizeEvent.off(this.targetEl, this.onTargetResize)
     window.removeEventListener(`mousedown`, this.checkForOuterAction)
     window.removeEventListener(`touchstart`, this.checkForOuterAction)
+    window.removeEventListener(`touchstart`, this.checkForOuterAction)
```
Comment (iakovmarkov) [inline src/index.js]: Duplicate?

File: src/layout.js [hunk 4]
```diff
@@ -223,11 +224,12 @@ const pickZone = (opts, frameBounds, targetBounds, size) => {
 }
 
 
-
 /* TODO Document this. */
 
-const calcRelPos = (zone, masterBounds, slaveSize) => {
+/* calculates position of the body element */
+const calcRelPos = (zone, masterBounds, slaveSize, parent) => {
```
Comment (iakovmarkov) [inline src/layout.js]: Is `parent` used here?

**Comment by codinglawyer** [comment]
Nope, since the body element is removed from the DOM immediately.

**Comment by iakovmarkov** [comment]
Great!

Now, please add the description of this new prop to README and we can merge this.

**Pull request merged**